### ISSUE TITLE
training code for DCVC-FM

### ISSUE THREAD

**Issue by xzhu0801** [microsoft/DCVC#72]
Hello, I want to reproduce the paper DCVC-FM. But I I often encountered the problem of **gradient explosion**.  Who can share me the training code please sent to [xzhu0801@gmail.com](url) or we can discuss together, thank you very much!!!

**Comment by gdit-ai** [comment]
Who can share me the training code please sent to 81664352@qq.com or we can discuss together, thank you very much!!!

**Comment by n3su** [comment]
Hello, I am currently training DCVC-FM. I was wondering if you could share training code or discuss it together?
This is my email : luyt216@gmail.com
Thank you very much!!!

**Comment by lzh1819** [comment]
This is my email : lzh1819@126.com thank you very much!!!

**Comment by yaohualibin** [comment]
We have released DCVC-UF code, including the training code. Please have a try.
